### DISCUSSION TITLE
有没有快速测试的方法

### DISCUSSION THREAD

**Discussion by yesy6sysu** [arcsysu/SYsU-lang#24] [Q&A]
在跑测试脚本的时候每次都需要重复测试lexer，有木有不测lexer直接测parser的方法呀

**Comment by wu-kan** [comment]
[RTFM](https://github.com/arcsysu/SYsU-lang/tree/main/parser#%E8%87%AA%E5%8A%A8%E8%AF%84%E6%B5%8B%E7%BB%86%E5%88%99)
